astro.config.mjs

```diff
@@ -12,8 +12,6 @@ export default defineConfig({
     Sentry({
       dsn: process.env.PUBLIC_ENV === "prod" ? process.env.PUBLIC_SENTRY_DSN : "",
       debug: true,
-      replaysSessionSampleRate: 0.1,
-      replaysOnErrorSampleRate: 1.0,
```

Comment:
options in this file are ignored on the client (on the server) as soon as you register a `sentry.client.config.js` (`sentry.server.config.js`) file. So to avoid further confusion, I removed the replay sample rates here.